src/citra_qt/main.cpp

```diff
@@ -1410,6 +1416,28 @@ void GMainWindow::OnRemoveAmiibo() {
     ui.action_Remove_Amiibo->setEnabled(false);
 }
 
+void GMainWindow::OnMenuSelectEmulatedDirectory(const FileUtil::UserPath path) {
+    const auto res = QMessageBox::information(this, tr("Changing Emulated Directory"),
+                                              tr("You are about to change the emulated %1 directory of the system. Please note "
```

Comment:
Just a nitpick, from a translator standpoint, when this sentence is taken out of the context (it is hard to know the origin of the string on Transifex), it would be hard to understand what the `%1` interpolates. I suggest to add an [assistive comment](https://doc.qt.io/qt-5/i18n-source-translation.html#translator-comments) above that line like this:

```suggestion
                                              //: Translators: possible values for %1: "SD card" or "NAND"
                                              tr("You are about to change the emulated %1 directory of the system. Please note "
```

I kind of want to apply this technique to other parts of the project where deducing the contents of interpolation is pretty difficult for the translators